Providers/Profile.swift

```diff
@@ -703,15 +703,22 @@ public class BrowserProfile: Profile {
 
         // Used as a task queue for syncing.
         private var syncReducer: AsyncReducer<EngineResults, EngineTasks>?
+        private var syncStatsReport: SyncStatsReport?
 
         private func beginSyncing() {
             notifySyncing(NotificationProfileDidStartSyncing)
+            if let account = profile.account {
+                syncStatsReport = SyncStatsReport(when: NSDate.now(), account: account, didLogin: false, why: "")
```

Comment:
That would be ideal. How would you suggest getting one? Is there a 'start' ping of some sort that can give me a timestamp?